src/crepe/api/GameObject.cpp

```diff
@@ -7,20 +7,17 @@
 using namespace crepe;
 using namespace std;
 
-GameObject::GameObject(ComponentManager & component_manager, game_object_id_t id,
-					   const std::string & name, const std::string & tag,
-					   const vec2 & position, double rotation, double scale)
+GameObject::GameObject(Mediator & mediator, game_object_id_t id, const std::string & name,
+					   const std::string & tag, const vec2 & position, double rotation,
+					   double scale)
```

Comment:
double scale?

Reply:
Mijn PR verandert alleen het 1e argument van ComponentManager naar Mediator. Er komen op meer plaatsen nog doubles inplaats van floats voor dus ik heb een miro ticket aangemaakt dat hier nog op gecontroleerd moet worden, maar ik laat het in deze PR zo staan.